source/core/index.ts

```diff
@@ -574,7 +574,9 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 						headers['content-type'] = encoder.headers['Content-Type'];
 					}
 
-					headers['content-length'] = encoder.headers['Content-Length'];
+					if ('Content-Length' in encoder.headers) {
+						headers['content-length'] = encoder.headers['Content-Length'];
+					}
```

Comment:
Why the `if`?

Reply:
Otherwise this header will be `undefined`, because the encoder does not include this header and I will get a `400 (Bad Request)` error in tests.